projects/1.19/assets/sgjourney/sgjourney/lang/zh_cn.json

```
@@ -0,0 +1,537 @@
+	"tooltip.sgjourney.crystal.in_dhd.energy.advanced": "将星门的目标能源容量提高100000FE",
+	"tooltip.sgjourney.crystal.in_dhd.transfer": "将传输到星门的能量增加2500FE",
+	"tooltip.sgjourney.crystal.in_dhd.transfer.advanced": "将传输到星门的能量增加5000FE",
+	"tooltip.sgjourney.crystal.in_dhd.communication": "将DHD与星门的通信距离增加16格",
+	"tooltip.sgjourney.crystal.in_dhd.communication.advanced": "将DHD与星门的通信距离增加32格",
+	"tooltip.sgjourney.communication_crystal.frequency": "频率",
+	"tooltip.sgjourney.materialization_crystal.increased_range": "增大范围",
+	"tooltip.sgjourney.materialization_crystal.interdimensional": "实现跨维度传输",
+
+	"tooltip.sgjourney.call_forwarding_device.in_dhd": "在DHD：",
+	"tooltip.sgjourney.call_forwarding_device.info": "启用呼叫转接",
+	
+	"tooltip.sgjourney.stargate_upgrade.description": "右击经典星门进行升级",
+	"tooltip.sgjourney.stargate_variant.description": "右击星门更改其变体",
+	
+	"tooltip.sgjourney.zpm.entropy": "熵",
+	"tooltip.sgjourney.zpm.energy_in_level": "能量等级",
+	
+	"tooltip.sgjourney.energy_transfer": "最大能量传输",
+	"tooltip.sgjourney.matok.open": "开启",
+	"tooltip.sgjourney.matok.closed": "关闭",
+	"tooltip.sgjourney.matok.open_close": "在主手Shift右击以打开/关闭",
+	"tooltip.sgjourney.matok.reload": "在副手Shift右击以重新装入主手的硅岩量瓶",
+	"tooltip.sgjourney.kara_kesh.terror": "模式：恐怖",
+	"tooltip.sgjourney.kara_kesh.knockback": "模式：击退",
+	"tooltip.sgjourney.kara_kesh.terror_knockback": "在主手Shift右击以更改模式",
+	"tooltip.sgjourney.kara_kesh.use":"对实体右击以使用",
+	"tooltip.sgjourney.pda.info": "右击获取目标信息",
+	"tooltip.sgjourney.syringe.contents": "目录",
+	"tooltip.sgjourney.syringe.empty": "空",
+	"tooltip.sgjourney.syringe.blood": "血",
+	"tooltip.sgjourney.syringe.prototype_ata_gene": "古代基因疗法原型",
+	"tooltip.sgjourney.syringe.ata_gene": "古代基因疗法",
+	
+	"message.sgjourney.ring_remote.error.transport_rings_busy": "错误：传送环被占用",
+	"message.sgjourney.ring_remote.error.no_transport_rings_nearby": "错误：附近没有传送环",
+	"message.sgjourney.ring_remote.error.no_coordinates": "错误：无坐标",
+	
+	"message.sgjourney.syringe.got_ancient_gene": "你现在拥有了古代基因",
+	"message.sgjourney.syringe.has_ancient_gene": "你已经拥有了古代基因",
+	"message.sgjourney.pda.has_ancient_gene": "拥有古代基因",
+	"message.sgjourney.pda.has_naquadah_in_bloodstream": "血液中含有硅岩",
+	
+	"message.sgjourney.dhd.error.not_connected_to_stargate": "DHD未连接到星门",
+	
+	"message.sgjourney.stargate.error.one_way_wormhole": "无法从这个方向通过虫洞",
+	
+	"message.sgjourney.stargate.info.none": "无",
+	"message.sgjourney.stargate.error.unknown": "未知错误",
+	
+	"message.sgjourney.stargate.info.symbol_encoded": "符号编码",
+	"message.sgjourney.stargate.error.symbol_in_address": "符号已编码",
+	"message.sgjourney.stargate.error.symbol_out_of_bounds": "符号越界",
+	"message.sgjourney.stargate.error.encode_when_connected": "连接时无法编码符号",
+	
+	"message.sgjourney.stargate.info.connection_established.system_wide": "已建立全系统连接",
+	"message.sgjourney.stargate.info.connection_established.interstellar": "建立星际联系",
+	"message.sgjourney.stargate.info.connection_established.intergalactic": "建立星系际联系",
+	

```

```suggestion
	"message.sgjourney.stargate.info.connection_established.system_wide": "已建立行星际连接",
	"message.sgjourney.stargate.info.connection_established.interstellar": "已建立星际连接",
	"message.sgjourney.stargate.info.connection_established.intergalactic": "已建立星系际连接",
```
很抱歉我在初次审核时没有注意时态和一致性问题